include/openssl/ec_key.h

```diff
@@ -356,6 +328,82 @@ OPENSSL_EXPORT EC_KEY *o2i_ECPublicKey(EC_KEY **out_key, const uint8_t **inp,
 OPENSSL_EXPORT int i2o_ECPublicKey(const EC_KEY *key, unsigned char **outp);
 
 
+// EC_KEY_METHOD
+
+// ECDSA_FLAG_OPAQUE specifies that this EC_KEY_METHOD does not expose its key
+// material. This may be set if, for instance, it is wrapping some other crypto
+// API, like a platform key store. Use |EC_KEY_METHOD_set_flag| to set
+// this flag on an |EC_KEY_METHOD|.
+// This was supported in ECDSA_METHOD previously and is not set by default.
+#define ECDSA_FLAG_OPAQUE 1
+
+// EC_KEY_get_default_method returns a reference to the default
+// |EC_KEY| implementation. The returned |EC_KEY_METHOD| object is
+// zero-initialized. This is different from OpenSSL which returns function
+// pointers to the default implementations within the |EC_KEY_METHOD| struct.
+// We do not do this to make it easier for the compiler/linker to drop unused
+// functions. The wrapper functions will select the appropriate
+// |ec_key_default_*| implementation.
+OPENSSL_EXPORT const EC_KEY_METHOD *EC_KEY_get_default_method(void);
```

Comment:
Valid points - I'll update documentation :)

Some context on OpenBSD documentation:
This API has another function EC_KEY_set_default_method that allows consumers to change the default method that future EC_KEY's will be created with. This is why the documentation for EC_KEY_get_default_method in OpenBSD indicates it returns the method "used for creation of new EC_KEY objects in the future". In our case, since we are not supporting this functionality yet, the default method is always returned. We can always change the internal implementation details later should we choose to support setting the default method.